contrib/bash/goshy

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -x
```

Comment:
Why printing the tracing output?

Reply:
Welp, … I just forgot to remove it. Resolved with 8cb43602bbe9cd1300b9855adab62a342319b308.